client/layout/guided-tours/tours/checklist-site-icon-tour.js

```diff
@@ -97,8 +104,8 @@ export const ChecklistSiteIconTour = makeTour(
 		>
 			<Continue target="image-editor-button-done" step="finish" click>
 				{ translate(
-					'Let’s make sure it looks right before you press {{b}}Done{{/b}} to save your changes.',
-					{ components: { b: <strong /> } }
+					'Let’s make sure it looks right before you press {{doneButton/}} to save your changes.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Let’s make sure it looks right before you press {{DoneButton/}} to save your changes.' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).